### DISCUSSION TITLE
sfcrmapps.cn and sfcrmproducts.cn sites

### DISCUSSION THREAD

**Discussion by jtryan0601** [tprouvot/Salesforce-Inspector-reloaded#330] [Q&A] [answered]
Does anyone know why these permissions are needed?

**Read and change your data on all sfcrmapps.cn sites and all sfrcrmproducts.cn sites**

Note: If my understanding is correct, .cn is the top-level domain for China.

**Comment by tprouvot** [comment, marked as answer]
Hi @jtryan0601,

Those are the new domains used by Salesforce hosted by AlibabaCloud for China.
This has been discussed in this topic:

https://github.com/tprouvot/Salesforce-Inspector-reloaded/discussions/326
